core/src/templates/components/card/card.twig

```diff
@@ -133,13 +143,14 @@
     {% endif %}
   {% endblock block_card_media -%}
 
-  <section class="su-card__contents">
+  {%- if card_contents_not_empty -%}
+  <div class="su-card__contents">
     {# Card Super Headline #}
-    {%- block block_card_super_headline %}
-      {% if card_super_headline is not empty %}
+    {% if card_super_headline is not empty %}
+      {%- block block_card_super_headline %}
```

Comment:
Blocks should wrap the conditional content statement. Blocks are not conditional in twig templates and will always be replaced/rendered if overridden from a parent template.

Reply:
If left in this order, a parent template would have to assign set card_super_headline to something truthy AND override the block statement with their logic.